inference-engine/tests/functional/plugin/conformance/test_runner/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME conformanceTests)
+
+list(APPEND EXPORT_DEPENDENCIES
+        gflags
+        funcSharedTests
+    )
+
+addIeTargetTest(
+        NAME ${TARGET_NAME}
+        ROOT "${CMAKE_CURRENT_SOURCE_DIR}/include"
+        ADDITIONAL_SOURCE_DIRS
+            ${CMAKE_CURRENT_SOURCE_DIR}/src
+        ADD_CPPLINT
+        INCLUDES
+            PUBLIC
+                "${CMAKE_CURRENT_SOURCE_DIR}/include"
+        LINK_LIBRARIES
+            PUBLIC
+                ${EXPORT_DEPENDENCIES}
+        DEPENDENCIES
+            ${EXPORT_DEPENDENCIES}
+        LABELS
+            CONFORMANCE
+)
+
+ie_faster_build(${TARGET_NAME}
+        PCH PRIVATE "src/precomp.hpp"
+        )
+
+if (TARGET MKLDNNPlugin)
```

Comment:
This really confuses me, why we need this dependency? And Why only MKLDNN?

Reply:
copy-paste issue. Removed. Thanks